Sources/SFSafeSymbols/Symbols/Localizable.swift

```diff
@@ -0,0 +1,329 @@
+// Don't touch this manually, this code is generated by the SymbolsGenerator helper tool
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+internal class LocalizableSymbol: SFSymbol, ar, he, hi, rtl, ja, ko, th, zh, zhTraditional, ar_v20, he_v20, hi_v20, rtl_v20, ja_v20, ko_v20, th_v20, zh_v20, zhTraditional_v20, ar_v21, he_v21, hi_v21, rtl_v21, ja_v21, ko_v21, th_v21, zh_v21, zhTraditional_v21, ar_v22, he_v22, hi_v22, rtl_v22, ja_v22, ko_v22, th_v22, zh_v22, zhTraditional_v22, ar_v30, he_v30, hi_v30, rtl_v30, ja_v30, ko_v30, th_v30, zh_v30, zhTraditional_v30, ar_v31, he_v31, hi_v31, rtl_v31, ja_v31, ko_v31, th_v31, zh_v31, zhTraditional_v31 {
+    var ar: SFSymbol { .init(rawValue: "\(rawValue).ar") }
+    var he: SFSymbol { .init(rawValue: "\(rawValue).he") }
+    var hi: SFSymbol { .init(rawValue: "\(rawValue).hi") }
+    var rtl: SFSymbol { .init(rawValue: "\(rawValue).rtl") }
+    var ja: SFSymbol { .init(rawValue: "\(rawValue).ja") }
+    var ko: SFSymbol { .init(rawValue: "\(rawValue).ko") }
+    var th: SFSymbol { .init(rawValue: "\(rawValue).th") }
+    var zh: SFSymbol { .init(rawValue: "\(rawValue).zh") }
+    var zhTraditional: SFSymbol { .init(rawValue: "\(rawValue).zh.traditional") }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol ar {
+    var ar: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol he {
+    var he: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol hi {
+    var hi: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol rtl {
+    var rtl: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol ja {
+    var ja: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol ko {
+    var ko: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol th {
+    var th: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol zh {
+    var zh: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol zhTraditional {
+    var zhTraditional: SFSymbol { get }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public protocol ar_v20 {
```

Comment:
Should this match how SPM handles macOS versions?

```suggestion
public protocol ar_v2_0 {
```

(or `_v2` for .0 releases and `_v2_2` for point releases?

Reply:
I'm open on how we do this. I just find `_v20` the most beautiful, I can't look at all of those underscores :D

Reply:
How about `_v2` for major versions and `_v21` for minors?

Reply:
The labeling isn't really future-proof, we'll have to rethink it in 7 years when v10 comes out.

Reply:
Then let's go with the initial suggestion of @j-f1!